content/docs/guides/logical-replication-decodable.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Replicate data with Decodable
+subtitle: Learn how to replicate data from Neon with Decodable
+enableTableOfContents: true
+isDraft: false
+updatedOn: '2024-07-05T15:15:00.000Z'
+---
+
+<LRNotice/>
+
+Neon's logical replication feature allows you to replicate data from your Neon Postgres database to external destinations.
+
+[Decodable](https://www.decodable.co/) is a fully managed platform for ETL, ELT, and stream processing,
+powered by Apache Flink® and Debezium.
+
+In this guide, you will learn how to configure a Postgres source connector in Decodable for ingesting changes from your Neon database so that you can replicate data from Neon to any of Decodable's supported data sinks,
```

Comment:
```suggestion
In this guide, you will learn how to configure a Postgres source connector in Decodable for ingesting changes from your Neon database so that you can replicate data from Neon to any of Decodable's [supported data sinks](https://docs.decodable.co/connect/destinations.html),
```